Add FAQ Section (#322)
helium/docs: docs/faq/build-on-network.mdx

---
id: build-on-network 
hide_title: true
title: Build On Network FAQ
sidebar_label: Build On Network
slug: /faq/build-on-network
---

# FAQ - Build On Network

> ### I want to use the Helium Network for my IoT project, how do I learn more?

Reviewer: ```suggestion
### I want to use the Helium Network for my IoT project. How do I learn more?
```

> Thanks for your interest! Check out or join the community Discord server as there are several channels dedicated to development. There’s also a ton of documentation on development and use for you, ready to peruse here: https://docs.helium.com/use-the-network

Reviewer: ```suggestion
Thanks for your interest! Join the [community Discord server](https://discord.gg/helium) as there are several channels dedicated to development. There’s also [extensive documentation on using the Network](https://docs.helium.com/use-the-network). 
```

### Can end devices communicate directly with each other or do they have to hop over the Helium Hotspot?

> All compatible devices communicate via hotspots. There is no direct communication or meshing in our network.

Reviewer: ```suggestion
All compatible devices communicate via Hotspots. There is no direct communication or meshing in the Helium Network.
```

### Will Helium be proprietary?

> We believe proprietary protocols and technologies stifle innovation. This is why we use commodity, open-standard hardware, and modules available from dozens of vendors. Even our software is open-sourced. Check out our Open-Sourced documentation.

Reviewer: ```suggestion
We believe proprietary protocols and technologies stifle innovation. This is why we use commodity, open-standard hardware, and modules available from dozens of vendors. And all software used is [open source under friendly licenses](https://docs.helium.com/open-source/open-source/#helium-is-open-source).
```

### Which frequencies does the Helium Hotspot use? Are you evaluating other frequencies in different countries?

> We use sub-ghz unlicensed spectrum which varies country by country.  You can also view the frequency used per country here: https://docs.helium.com/lorawan-on-helium/frequency-plans/

Reviewer: ```suggestion
Hotspots create LoRaWAN coverage, which sub-ghz unlicensed spectrum that varies country by country.  You can [view the frequency used per country here](https://docs.helium.com/lorawan-on-helium/frequency-plans).
```

> ### What kind of data rates do you anticipate going from device to the hotspot?

Reviewer: ```suggestion
### What kind of data rates do you anticipate going from device to the Hotspot?
```

> The applications we’re targeting for traffic typically within 5 kbps.

Reviewer: ```suggestion
Applications that are suitable for the Helium Network send small amounts of data. The maximum amount possible is around 5kbits per second. But this is rarely seen in production. Typically sensors are sending 10-1000 bytes at a time.
```
